ajax/libs/chartjs-plugin-zoom/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "chartjs-plugin-zoom",
+  "filename": "chartjs-plugin-zoom.min.js",
+  "author": {
+    "name": "chartjs",
+    "url": "https://github.com/chartjs"
+  },
```

Comment:
It is not the author's information.
Please correct `name` and `url` fields, thanks!